README.md

```diff
@@ -296,7 +296,7 @@ The OpenTelemetry Operator *might* work on versions outside of the given range,
 
 | OpenTelemetry Operator | Kubernetes           | Cert-Manager         |
 |------------------------|----------------------|----------------------|
-| v0.51.0                | v1.19 to v1.23       | 1.6.1                |
+| v0.51.0                | v1.19 to v1.23       | 1.8.0                |
```

Comment:
This should not be changed either. The new release will start using the 1.8.0.

I want to raise a slightly different message here. We should probably put here the kind version instead of the cert-manager version because we should be compatible with the cert manger kinds. I would love to see if the oldest version of the cert-manger works with OTEL operator.